cpg-core/src/main/java/de/fraunhofer/aisec/cpg/TranslationConfiguration.java

```diff
@@ -448,6 +448,7 @@ public Builder defaultPasses() {
       registerPass(new TypeResolver());
       registerPass(new ControlFlowSensitiveDFGPass());
       registerPass(new FilenameMapper());
+      registerPass(new StatisticsCollectionPass());
```

Comment:
Don't add this as default pass. Leave it to users to decide if they require these metrics.